ui/scss/component/section.scss

```diff
@@ -1,4 +1,5 @@
 .section {
+  position: relative;
```

Comment:
why is this necessary?

Reply:
it's good to avoid changing from position static when feasible because it avoids introducing additional stacking contexts (which is why everyone writes shit z-index css)